uvu-tests/lint-teleport-docs-links.test.ts

```diff
@@ -0,0 +1,110 @@
+import { suite } from "uvu";
+import * as assert from "uvu/assert";
+
+import { VFile, VFileOptions } from "vfile";
+import { remark } from "remark";
+import mdx from "remark-mdx";
+import { remarkLintTeleportDocsLinks } from "../server/lint-teleport-docs-links";
+
+const transformer = (options: VFileOptions) => {
+  const file = new VFile(options);
+
+  return remark().use(mdx).use(remarkLinks).processSync(file).toString();
```

Comment:
Where is `remarkLinks` supposed to come from? Why is it showing up as undefined

Reply:
`transformer` was left over from when I had combined this linter with `remarkLinks`. I've removed `transformer`